src/features/community/project-dignity/components/FrogContainer.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import frogStatue from "assets/nfts/frogs/frog_statue.png";
+import { GRID_WIDTH_PX } from "features/game/lib/constants";
+import { Frog } from "../models/frog";
+import { FrogComponent } from "./FrogComponent";
+
+interface Props {
+  frogs: Frog[];
+  farmId: number | undefined;
+}
+
+export const FrogContainer: React.FC<Props> = ({ frogs, farmId }) => {
+  return (
+    <div
+      style={{
+        width: `${GRID_WIDTH_PX * 12}px`,
+        left: `${GRID_WIDTH_PX * 14}px`,
+        top: `${GRID_WIDTH_PX * 2.2}px`,
+      }}
+      className="flex absolute justify-center"
+    >
+      {farmId === 143560 && (
```

Comment:
What does this hardcoded farmId represent?

Reply:
There was one frog that somehow was sent to a farm address, so now it is "burned". Farm id is hardcoded because we dont want to incentivize burning of frogs and this is one of a kind statue on that particular farm.